### ISSUE TITLE
VV quadrupled its memory footprint since VV 1.3

### ISSUE THREAD

**Issue by SimonRit** [open-vv/vv#48]
Opening a 3.3 GB image takes 17 GB of RAM in the latest version.

**Comment by djboersma** [comment]
I wonder how you know that it duplicates the images. Did you find the part of the code where that happens? I also wonder if it matters what type the image is (type of pixel value), and whether VV maybe converts it to a different type. If the image on disk has unsigned chars but you read it as normal unsigned ints then you also get this multiplication factor. And finally I wonder if the image on disk is maybe compressed in any way.

In other words: I think that it would be useful to get more specs.

**Comment by SimonRit** [comment]
Good points. Short answer: I'm not sure this is the problem but my gut feeling is there is a good chance this is the reason. In the history of the dev of VV, we had made sure at some point that an image was not duplicated in memory. The same 3.3 GB image opened with vv 1.3 takes less than 4 GB in memory. I'm not sure what caused the change in the many recent dev so I have changed the title.
For the rest: my image is float, not compressed.

**Comment by tbaudier** [comment]
The problematic commit is 10a10f801f12d2cea51dd564a67003971ad66269

"Debug the use of transformation matrix .mat"
I have to investigate because I don't remind me why I did that. (It seems there is no effect on the transform matrix)
